mlonmcu/target/riscv/etiss.py

```diff
@@ -803,6 +817,8 @@ def get_platform_defs(self, platform):
         ret["MEM_ROM_LENGTH"] = self.rom_size
         ret["MEM_RAM_ORIGIN"] = self.ram_start
         ret["MEM_RAM_LENGTH"] = self.ram_size
+        ret["MEM_FLASH_ORIGIN"] = self.flash_start
+        ret["MEM_FLASH_LENGTH"] = self.flash_size
```

Comment:
Shouldn't there be related `mlonmcu-sw` changes, too? Could you open up a PR for that as well?